frontend/packages/kubevirt-plugin/src/components/dashboards-page/vm-dashboard/vm-details-card.tsx

```diff
@@ -66,6 +77,9 @@ export const VMDetailsCard: React.FC<VMDetailsCardProps> = () => {
           <DetailItem title="Created" error={false} isLoading={!vmiLike}>
             <Timestamp timestamp={getCreationTimestamp(vmiLike)} />
           </DetailItem>
+          <DetailItem title="Hostname" error={!hostname} isLoading={!vmiLike}>
+            {hostname}
```

Comment:
just a though: what about the hostname of offline VM? Do we want to show it, even though there might be discrepancies?

Reply:
yes, we want users to know the can get more data if they install the guest agent, @matthewcarleton please keep me honest here.

Ref: http://openshift.github.io/openshift-origin-design/designs/virtualization/4.x/expose-guest-data/expose%20guest%20data.html#vms-overview-tab--details-card

Reply:
Good question. Kobe's correct that the design shows it as hidden when the GA isn't installed. I suppose we could show the offline VM hostname, but we may want to add some sort of indicator with a tooltip that explains that there may be discrepancies and to install the GA to get more accurate information. Let's see what @matthewcarleton has to say.

Reply:
so for the case where the GA isn't installed we could show the 
```
Hostname
Guest agent required
```
Is that what we're talking about here? I like the idea of not hiding it b/c it teaches the user about the thing they need.

Reply:
This thread is about using information like hostname from other sources than GA. We can harvest this info from VM spec or from cloud-init. This info can be used even when the VM is not running or GA is unavailable. 

The main problem is that the value might be different than the real one in the running VM.

Should we show the information and show some explanation or not show it at all?

Reply:
Ahh, I didn't understand the question.
NO , IMHO we should not show hostname if we don't know in a reliable way.

Notes:
a - we have no reliable way to shoe the hostname if the VM have no running guest agent supporting hostname queries.
b - the design goal is to have users install GA, we actively want users to see that the data is "Not Available" if no GA installed.

Reply:
ah - I agree we shouldn't show info that is inaccurate (or has the potential to be)
We should show what I have above when no GA is detected.

Reply:
I have updated the fields in the Overview tab Details card and the Details tab to show 'Guest agent required' when the GA isn't installed and 'Not available' when the GA is installed, but the data hasn't been loading yet. Should we show something like 'Guest agent loading', instead?